drivers/source/serial_io.c

```diff
@@ -24,7 +24,10 @@ void sciMutexInit(void) {
     }
```

Comment:
Looks like there are some build warnings that need to be fixed. Ignore the ones related to the `sys_selftest.c` file.